### PR TITLE
[BT-658] Adjust components for InactiveUserView

### PR DESCRIPTION
**Pull request by ahollesen**
- reusing background-color for body
- added isTargetBlank prop to link + example

### CODE REVIEW

**Comment by jenslonkowski** [comment]
Spotless! 👍

File: src/framework/link/Link.jsx [hunk 1]
```diff
@@ -28,6 +34,7 @@ Link.propTypes = {
   href: PropTypes.string,
   onClick: PropTypes.func,
   data: PropTypes.any,
+  isTargetBlank: PropTypes.bool,
```
Comment (cjcenizal) [inline src/framework/link/Link.jsx]: I think we should leverage convention for components which are simple wrappers around HTML elements. So I think we should just make this `target` instead, and require the dev to pass in `"_blank"`.

**Comment by cjcenizal** [comment]
1 comment, then 👍

File: src/framework/link/Link.jsx (hunk 1 above)
Comment (ahollesen) [inline src/framework/link/Link.jsx]: Actually that was my first version, but then I thought that any other than `"_blank"` wouldn't be a proper option, as they are rarely or actually never used. I'm fine with both, the call is yours :)

Comment (cjcenizal) [inline src/framework/link/Link.jsx]: I hear you, I think we should go with my suggestion. At some point we might want to target specific tabs (http://stackoverflow.com/questions/6903815/how-to-open-link-url-in-specific-browser-tab-window-page).

**Pull request merged**